docs/tutorials/tokens/erc20.md

```diff
@@ -214,9 +252,100 @@ burn it, but a different contract may define its own permission model.
 }
 ```
 
-> **NOTE:** When burning a non-fungible token, the amount must always be `1`. The `tokenIndex` field is also required when burning a non-fungible token.
-
 Other parameters:
 - You must specify a `pool` name if you've created more than one pool
 - You may specify a `key` understood by the connector (i.e. an Ethereum address) if you'd like to use a non-default signing identity
 - You may specify `from` if you'd like to burn tokens from a specific identity (default is the same as `key`)
+
+
+## Token approvals
+You can also approve other wallets to transfer tokens on your behalf with the `/approvals` API. The important fields in a token approval API request are as follows:
+
+- `approved`: Sets whether another account is allowed to transfer tokens out of this wallet or not. If not specified, will default to `false`.
```

Comment:
This is accurate. But do you think it's too confusing and worthy of changing in the code?

Reply:
I think it's worthy of changing the code. But until then, I think we need to call it out in the docs

Reply:
https://github.com/hyperledger/firefly/pull/782

Reply:
I will update this accordingly now